src/util/ChartUtils.ts

```diff
@@ -437,27 +380,41 @@ export const getBarPosition = ({
   return result;
 };
 
-export const appendOffsetOfLegend = (offset: any, items: Array<FormattedGraphicalItem>, props: any, legendBox: any) => {
+export const appendOffsetOfLegend = (
+  offset: ChartOffset,
+  _unused: unknown,
```

Comment:
if `items` was unused could we not just remove it?

Reply:
That could be a breaking change.

Reply:
How? I guess I generally wouldn't expect anyone externally to be using this function and if they are it certainly isn't documented. Would say semver doesn't really apply to helper functions (doesn't help that we export literally everything but still)

Reply:
Perhaps the next major change in recharts should be to stop exporting things and have a clear distinction between private and public APIs

Reply:
Keep or remove for this one I guess doesn't matter to me, I would argue if someone breaks it was their risk to accept (and in this case the fix is easy). 

But yes completely agree, that helps us with maintainability and even allows us to export some non-component functions that may be helpful and document them.